Task/Runner/Nuke/Build.cs

```diff
@@ -178,6 +178,42 @@ public class Build : NukeBuild
             }
         });
 
+    Target TestServerE2e => _ => _
+        .Description("Run E2E Playwright tests using Docker Compose")
+        .DependsOn(DbResetForce)
+        .Executes(() =>
+        {
+            if (Directory.Exists(TestResultsDirectory))
```

Comment:
As per my other comment we should put the trx reports for the e2e tests in Reports/e2e instead, so they don't overlap with the existing xunit trx reports. Don't touch the xunit trx paths as we will refactor them in a separate PR.